packages/jest-dom-mocks/src/storage.ts

```diff
@@ -7,14 +7,14 @@ export default class Storage {
 
   clear = jest.fn(this.unmockedClearItem);
 
+  key = jest.fn(this.unmockedKey);
+
+  length = jest.fn(this.unmockedLength);
+
   private store: {
     [key: string]: string;
   } = Object.create(null);
 
-  get length() {
```

Comment:
Length needs to remain a getter because it is a property, not a method. You do `storage.length` not `storage.length()`